core/src/main/java/com/xmartlabs/bigbang/core/module/OkHttpModule.kt

```diff
@@ -73,13 +73,9 @@ open class OkHttpModule {
 
     try {
       val statFs = StatFs(dir.absolutePath)
-      val blockCount: Long
-      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-        blockCount = statFs.blockCountLong
-      } else {
-        @Suppress("DEPRECATION")
-        blockCount = statFs.blockCount.toLong()
-      }
+      @Suppress("DEPRECATION")
+      val blockCount = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2)
```

Comment:
I don't like extract it to a function, we should work in a better solution. (Some extension in the context or something like that)